modules/nw-networkpolicy-edit.adoc

```diff
@@ -16,18 +17,21 @@ endif::[]
 
 You can edit a {name} policy in a namespace.
 
-ifndef::multi[]
+ifndef::multi,microshift[]
 [NOTE]
 ====
 If you log in with a user with the `cluster-admin` role, then you can edit a network policy in any namespace in the cluster.
 ====
-endif::multi[]
+endif::multi,microshift[]
 
 .Prerequisites
-
+ifndef::microshift[]
 * Your cluster uses a network plugin that supports `NetworkPolicy` objects, such as the OVN-Kubernetes network plugin or the OpenShift SDN network plugin with `mode: NetworkPolicy` set. This mode is the default for OpenShift SDN.
+endif::microshift[]
 * You installed the OpenShift CLI (`oc`).
+ifndef::microshift[]
 * You are logged in to the cluster with a user with `{role}` privileges.
+endif::microshift[]
```

Comment:
Suggestions to L54:

** If you saved the {name} policy definition in a file, edit the file and make any necessary changes, and then enter the following command: